util/version/Cargo.toml

```diff
@@ -3,7 +3,7 @@
 [package]
 name = "parity-version"
 # NOTE: this value is used for Parity version string (via env CARGO_PKG_VERSION)
-version = "1.10.0"
+version = "1.10.1"
```

Comment:
We should actually unflag `critical` a bit below this

Reply:
Authorities should be running with `--auto-update` anyway. Marking it as critical will cause all Parity nodes to update (and it's a hassle to not deploy critical update on mainnet, we should actually have this flag separate between networks).